dist/htmlminifier.js

```diff
@@ -285,8 +237,30 @@
       parseEndTag();
     }
 
-    function parseStartTag( tag, tagName, rest, unary ) {
-      var unarySlash = false;
+    function parseStartTag(input) {
+      var start = input.match(startTagOpen);
+      if (start) {
+        var match = {
```

Comment:
Noted for `var` hoisting.

As for `let`/`const` - I don't think we are quite ready for ES6 yet if we are to support a range of web browsers...